packages/geany_plugins.rb

```diff
@@ -0,0 +1,33 @@
+require 'package'
+
+class Geany_plugins < Package
+  description 'Geany plugins.'
+  homepage 'https://www.geany.org/'
+  version '1.37'
+  compatibility 'aarch64,armv7l,x86_64'
+  case ARCH
+  when 'aarch64', 'armv7l', 'x86_64'
+    source_url 'https://download.geany.org/geany-plugins/geany-plugins-1.37.tar.bz2'
+    source_sha256 'c98f9b1303f4ab9bed7587e749cd0b5594d9136a1bf8ba110900d46a17fa9cd8'
+    depends_on 'gtk3'
+    depends_on 'xdg_base'
+    depends_on 'sommelier'
+    depends_on 'geany'
```

Comment:
Since geany already has `gtk3`, `xdg_base` and `sommelier` dependencies, no need to add here.

Reply:
adjusted accordingly